src-built-in/components/assets/css/_dialogs.css

```diff
@@ -80,25 +80,29 @@ body.dialog input::-webkit-outer-spin-button,
 }
 
 body.dialog input:focus {
-    box-shadow: 0 0 5px rgba 81, 203, 238, 1;
-    border: 1px solid var(--primary-accent-color);
+    border: 1px solid var(--dialog-input-border-focus-color);
     outline: none;
 }
 
 body.dialog input {
     -webkit-appearance: none;
-    color: var(--dialog-font-color);
+    color: #FFFFFF;
     text-align: left;
     font-size: 12pt;
+    font-family: Roboto;
```

Comment:
@DanNicolai why are we hard coding font-family and color instead of referencing variables?